Bug: Error: Cannot find module '@circleci/circleci-config-sdk/dist/src/lib/Components/Executors/exports/DockerImage'

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

```shell
yarn run v1.22.17
$ ts-node index.ts -f ../../.circleci/config.yml
Error: Cannot find module '@circleci/circleci-config-sdk/dist/src/lib/Components/Executors/exports/DockerImage'
Require stack:
- /myproject/scripts/generate_circleci_config/index.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (myproject/scripts/generate_circleci_config/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (myproject/scripts/generate_circleci_config/index.ts:2:1)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module.m._compile (myproject/scripts/generate_circleci_config/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.require.extensions.<computed> [as .ts] (myproject/scripts/generate_circleci_config/node_modules/ts-node/src/index.ts:1621:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'myproject/scripts/generate_circleci_config/index.ts'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```


### Minimum reproduction code

https://gist.github.com/mstva/d2abbae90e9d60b4d485386bbe2f5832

### Steps to reproduce

_No response_

### Expected behavior

it should give me the job like this

```ymal
version: 2.1
setup: false
jobs:
  run_unit_tests:
    docker:
      - image: cimg/python:3.10
      - image: cimg/postgres:13.8
```

### CircleCI Config SDK version

0.11.1

### Node.js version

18.11.0

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Other

_No response_

--- a/src/lib/Components/Executors/types/DockerExecutor.types.ts
+++ b/src/lib/Components/Executors/types/DockerExecutor.types.ts
@@ -1,5 +1,10 @@
 import { StringParameter } from '../../Parameters/types';
-import { DockerImageShape } from '../exports/DockerImage';
+import {
+  DockerImageShape,
+  DockerImage,
+  DockerAuth,
+  DockerAuthAWS,
+} from '../exports/DockerImage';
 import { AnyResourceClassBase } from './Executor.types';
 /**
  * A JSON representation of the Docker Executor Schema
@@ -16,3 +21,5 @@ export type DockerImageMap = {
  * @see {@link https://circleci.com/docs/2.0/configuration-reference/#docker-executor} for specifications of each class.
  */
 export type DockerResourceClass = AnyResourceClassBase;
+
+export { DockerImageShape, DockerImage, DockerAuth, DockerAuthAWS };

